deployment/README.md

```diff
@@ -213,6 +284,8 @@ should be sufficient.
 
 Fluentd is deployed with no replicas. Once you have ElasticSearch
 running as desired, scale fluentd to every node to feed logs into ES.
+The example below would be for a cluster with 3 nodes; substitute the
+actual number of nodes in your cluster.
 
     oc scale dc/logging-fluentd --replicas=3
```

Comment:
Strictly speaking we don't need to change the DC, as the RC is the actual deployment and determines how many pods are running. The DC only comes into it if you create a new deployment, which the user in the case of Fluentd/Kibana has no real reason to do (more of a development use case). The new deployment would get the scale of the DC, not the previous deployment. So to me it makes sense to keep them in sync. It felt like it would be more confusing to explain all this than not, but perhaps I'm wrong.